website/docs/d/auth_backend.html.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
You need to add a link to `website/vault.erb` or this page will not show up in the sidebar.

Reply:
Updated, thanks!